python/src/otel/otel_sdk/otel-instrument

```diff
@@ -96,7 +96,7 @@ fi
 # - Set the metrics exporter
 
 if [ -z "${OTEL_METRICS_EXPORTER}" ]; then
-    export OTEL_METRICS_EXPORTER=none;
+    export OTEL_METRICS_EXPORTER=otlp;
```

Comment:
should we avoid setting anything explicitly so that we rely on what is on the spec/sdk? https://opentelemetry.io/docs/reference/specification/sdk-environment-variables/#exporter-selection

Reply:
Not setting is more future proof than setting in the unlikely event of default is changed.

Reply:
I agree that this wrapper probably shouldn't change the default behavior in this way.

Reply:
Ok so setting this OTEL_METRICS_EXPORTER environment variable should be removed overall so the default behavior in the sdk can be used?